notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-797_analysis.ipynb

```diff
@@ -0,0 +1,1401 @@
+{
```

Comment:
<p>Please, add label to the vertical limit lines. if the reader go directly to see the figure, it is not clear what the lines are.</p><p> </p><p>Also, the title of the figure is too general. I think it should be something more meaningful about the specici analysis done here, instead of something more general related to the name of the ticket itsled.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Done</p>